src/elm/Pages/Build/Model.elm

```diff
@@ -33,10 +36,14 @@ import Vela
 type alias PartialModel a =
     { a
         | navigationKey : Navigation.Key
+        , user : WebData CurrentUser
+        , sourceRepos : WebData SourceRepositories
+        , page : Page
         , time : Posix
         , zone : Zone
         , repo : RepoModel
         , shift : Bool
+        , pipeline : PipelineModel
```

Comment:
adding some more data to `Build.Model` and `Pipeline.Model` so that they may be passed as a PartialModel to `Nav.elm`